katcbfsim/stream.py

```diff
@@ -877,6 +889,7 @@ def _baseline_correlation_products_sensors(self, view):
                         baselines.append((name1, name2))
         self.sensor(view, 'bandwidth', self.bandwidth)
         self.sensor(view, 'bls_ordering', baselines)
+        self.sensor(view, 'input_labels', input_labels)
```

Comment:
Why add input_labels?

Reply:
The `input_data_suspect` is an array of booleans, and `input_labels` tells us which input corresponds to each position in that array. I don't think katcbfsim produces `input_data_suspect` (yet), but ingest now requires `input_labels` to be present.